CMakeLists.txt

```diff
@@ -57,6 +57,14 @@ if (DEFINED CMAKE_SHARED_LINKER_FLAGS)
 	STRING(REPLACE "-Wl,--no-undefined" "" CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS}")
 endif (DEFINED CMAKE_SHARED_LINKER_FLAGS)
 
+
+if(UNIX)
```

Comment:
Normally the style is to have a space between the if and the (VAR), although there were inconsistent edits in the past.